scripts/bert/run_pretraining.py

```diff
@@ -263,13 +273,15 @@ def evaluate(data_eval, model, nsp_loss, mlm_loss, vocab_size, ctx):
                 local_mlm_loss += ls1.as_in_context(mx.cpu())
                 local_nsp_loss += ls2.as_in_context(mx.cpu())
                 local_num_tks += valid_length.sum().as_in_context(mx.cpu())
+
+                total_mlm_loss += local_mlm_loss
```

Comment:
This does not seem correct, `local_mlm_loss` is reset to 0 every args.log_interval iterations.

Reply:
You're right, I messed it up there. I reworked it a bit and hopefully made it clearer :)